index.html

```diff
@@ -35,6 +35,11 @@
     <div class="home-map" data-hash-association="#!/home" class="active"></div>
 
     <div class="bat-detective app">
+      <div class="world-tour-banner">
+        <h2>Bat Detective World Tour starts November 2 with the United Kingdom!</h2>
+        <p>Learn more at the <a href="http://blog.batdetective.org/" target="_blank">blog</a></p>
```

Comment:
Hi guys, Rory here from Bat Detective. In terms of final wording for the text, I think this should be fine: 

Bat Detective is on a World Tour! We're currently in the United Kingdom. Learn more at the <a href="http://blog.batdetective.org/2015/11/02/bat-detective-world-tour-uk-information/" target="_blank">blog</a>

Where on the front page will this banner be appearing? We also had a few other specific changes it'd be ideal to make to the Bat Detective main site if possible - is it best for me to raise these as separate issues on here? Thanks very much.